CMakeLists.txt

```diff
@@ -42,10 +42,12 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 set(ROOT_DIR "${CMAKE_CURRENT_LIST_DIR}")
 
-set(TITLE_SEQUENCE_URL  "https://github.com/OpenRCT2/title-sequences/releases/download/v0.1.2c/title-sequences.zip")
+set(TITLE_SEQUENCE_VERSION "0.1.2c")
+set(TITLE_SEQUENCE_URL  "https://github.com/OpenRCT2/title-sequences/releases/download/v${TITLE_SEQUENCE_VERSION}/title-sequences.zip")
 set(TITLE_SEQUENCE_SHA1 "304d13a126c15bf2c86ff13b81a2f2cc1856ac8d")
 
-set(OBJECTS_URL  "https://github.com/OpenRCT2/objects/releases/download/v1.0.21/objects.zip")
+set(OBJECTS_VERSION "1.0.21")
+set(OBJECTS_URL  "https://github.com/OpenRCT2/objects/releases/download/v${OBJECTS_VERSION}/objects.zip")
```

Comment:
Seems odd to split up the sequences and objects into version and url but not replays. I know you don't need the version as you don't use it in your bundle for replays but should we perhaps be consistent.

Reply:
@duncanspumpkin the inconsistency was bothering me a bit too, but just not enough to do anything about it. Thanks for pushing me over the edge :)

Check out the last two commits. I went ahead and switched the existing "does this folder/file exist?" downloads in the install steps to use the versioned download method. I also updated that method to be backward compatible with older versions of cmake (like those used in the linux CI image).